beamer/l1_resolution.py

```diff
@@ -0,0 +1,26 @@
+import subprocess
+
+from hexbytes import HexBytes
+
+from beamer.typing import URL
+
+RELAYER_EXECUTABLE = "beamer-relayer"
+
+
+def run_relayer(l1_rpc: URL, l2_rpc: URL, privkey: str, tx_hash: HexBytes) -> None:
+    subprocess.run(
+        [
+            RELAYER_EXECUTABLE,
+            "--l1rpcprovider",
+            l1_rpc,
+            "--l2rpcprovider",
+            l2_rpc,
+            "--l1wallet",
+            privkey,
+            "--froml2transactionhash",
```

Comment:
Nitpick: joining words makes for a harder-to-read option. Usually the convention is to use `-`, e.g. `--from-l2-transaction-hash`, `--l1-wallet`, `--l2-rpc-provider`...

Reply:
Pushed into https://github.com/beamer-bridge/beamer/issues/649